libgnucash/app-utils/options.scm

```diff
@@ -1721,7 +1721,13 @@
                                         "Use Full Other Account Name?" (cons #f "Use Full Other Account Name")
                                         "Void Transactions?" (cons "Filter" "Void Transactions")
                                         "Void Transactions" (cons "Filter" "Void Transactions")
-                                        "Account Substring" (cons "Filter" "Account Matcher")
+                                        "Account Substring" (cons "Filter" "Account Name Filter")
+                                        "Account Matcher" (cons "Filter" "Account Name Filter")
+                                        "Transaction Matcher" (cons "Filter" "Transaction Filter")
+                                        "Account Matcher uses regular expressions for extended matching"
+                                        (cons "Filter" "Use regular expressions for account name filter")
+                                        "Transaction Matcher uses regular expressions for extended matching"
+                                        (cons "Filter" "Use regular expressions for transaction filter")
```

Comment:
Why have you added all these in the option migrations ? Aren't these newly introduced options in master/unstable only ? Or did they exist in 2.6 already ?

Reply:
I'd thought they needed to be included because GST options using old section/name were in a previous 2.7.x release and should be migrated. However agree they're not in maint.